### PR TITLE
Ana/apply polkadotjs seed validation

### LINKED ISSUE #3951: Apply seed validation like the one from PolkadotJS for account import
It seems account import in Polkadot requires of some seed validation for account restore, otherwise it throws an error

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #3951

**Description:**

Applies exact same seed validation as seen in [polkadot ui](https://github.com/polkadot-js/apps/blob/4d17d5236d62c84009351b043993e53ee9fbd670/packages/page-accounts/src/Accounts/modals/Create.tsx).

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by Bitcoinera** [comment]
Thanks @mariopino for finding the piece of code! 🙏

**Comment by faboweb** [comment]
Can you add tests?

**Comment by Bitcoinera** [comment]
Tests added

File: src/components/common/TmSessionImport.vue [hunk 1]
```diff
@@ -25,12 +25,12 @@
           <TmFormMsg
             v-else-if="$v.seed.$error && !$v.seed.seedHasCorrectLength"
             name="Seed"
-            type="words12or24"
+            :type="isPolkadot ? 'incorrectPolkadotSeed' : 'words12or24'"
```
Comment (faboweb) [inline src/components/common/TmSessionImport.vue:28]: instead you could have used "custom"

**Pull request merged**